src/firecracker/src/main.rs

```diff
@@ -119,7 +119,7 @@ fn main_exitable() -> FcExitCode {
         .arg(
             Argument::new("id")
                 .takes_value(true)
-                .default_value(logger::DEFAULT_INSTANCE_ID)
+                .default_value("anonymous-instance")
```

Comment:
Why not use a `const` and use a string literal?

Reply:
It's only used here so making a constant only adds additional lines.